tests/derive/enum_unsupported.stderr

```diff
@@ -0,0 +1,5 @@
+error: derive(Bundle) does not support enums or unions
```

Comment:
Misleading error message

Reply:
That error message is for #[derive(Bundle)], which still only support structs
This PR only adds support for #[derive(Query)]

Reply:
Whoops, right.